### PR TITLE
Initializing project

### PR DESCRIPTION
**Pull request by tomastoloza**
things added:

typescript
typeorm
postgres
gitignore

using https://typeorm.io/#installation and https://typeorm.io/#quick-start

### CODE REVIEW

**Comment by tomastoloza** [comment]
[Crear el repositorio para el backend](https://trello.com/c/87wghK8f/24-crear-el-repositorio-para-el-backend)

**Comment by tomastoloza** [comment]
added dotenv library to manage credentials

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Awesome Project Build with TypeORM
+
+Steps to run this project:
+
```
Comment (adrianyaniri) [inline README.md:4]: ????

**Review by adrianyaniri — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (adrianyaniri) [inline README.md:4]: ??

**Review by adrianyaniri — CHANGES_REQUESTED** [review]

**Pull request merged**